core/src/epicli/data/common/ansible/playbooks/roles/filebeat/templates/filebeat.yml.j2

```diff
@@ -179,7 +179,12 @@ setup.template.settings:
 # These settings control loading the sample dashboards to the Kibana index. Loading
 # the dashboards is disabled by default and can be enabled either by setting the
 # options here or by using the `setup` command.
-setup.dashboards.enabled: false
+setup.dashboards.enabled: true
+
+# The Elasticsearch index name.
+# This setting overwrites the index name defined in the dashboards and index pattern.
+# Example: "testbeat-*"
+setup.dashboards.index: "filebeat-*"
```

Comment:
Have you tested a case when there are 2 (or more) Filebeat indices? By default they contain Filebeat version in the name so there are at least 2 after Filebeat's upgrade. Maybe this setting should be also configurable? And if yes, then it should be documented.